src/resources/views/crud/datatable/datatables_logic.blade.php

```diff
@@ -0,0 +1,520 @@
+@php
```

Comment:
Ok let's do it. Let's clean up this file a bit - let's take it step-by-step. First: let's remove the PHP from this file.

Reply:
Postponed